utils/tests/utils.go

```diff
@@ -96,7 +97,8 @@ func init() {
 	DockerRepoDomain = flag.String("rt.dockerRepoDomain", "", "Docker repository domain")
 	DockerVirtualRepo = flag.String("rt.dockerVirtualRepo", "", "Docker virtual repo")
 	DockerRemoteRepo = flag.String("rt.dockerRemoteRepo", "", "Docker remote repo")
-	DockerLocalRepo = flag.String("rt.DockerLocalRepo", "", "Docker local repo")
+	DockerLocalRepo = flag.String("rt.dockerLocalRepo", "", "Docker local repo")
+	DockerPromoteLocalRepo = flag.String("rt.dockerPromoteLocalRepo", "", "Docker promote local repo")
```

Comment:
Please document `rt.dockerPromoteLocalRepo` in the README
https://github.com/RobiNino/jfrog-cli/tree/tests-local-rt#docker-tests